common/primitives/src/msa.rs

```diff
@@ -16,6 +16,9 @@ pub use crate::schema::SchemaId;
 /// Message Source Id or msaId is the unique identifier for Message Source Accounts
 pub type MessageSourceId = u64;
 
+/// Maximum # of providers per delegator for deletion/benchmarking use
+pub const MAX_NUMBER_OF_PROVIDERS_PER_DELEGATOR: u32 = 128;
```

Comment:
maybe we pass this as a config into msa

Reply:
I had considered that but Wil and I decided to use a constant.  I can easily change it if you want.

Reply:
The rationale for using a constant is that there is no logic on `add_provider` to enforce it.  It’s just used for benchmarking.

Reply:
```
		/// The maximum number of providers per delegator
		#[pallet::constant]
		type MaxNumberOfProvidersPerDelegator: Get<u32>;
```